projects/1.18-fabric/assets/tempad/tempad/lang/zh_cn.json

```
@@ -0,0 +1,30 @@
+{
+  "gui.tempad.header_line_1": "任意门 ////",
+  "gui.tempad.header_line_2": "传送程序",
+  "gui.tempad.options": "1. 选项",
+  "gui.tempad.run_program": "2. 传送程序",
+  "gui.tempad.wiki": "3. Wiki",
+  "gui.tempad.textfield": "名称：",
+  "gui.tempad.new_location": "(+) 添加新坐标",
+  "gui.tempad.add_location": "添加坐标",
+  "gui.tempad.select_location": "选择坐标：",
+  "gui.tempad.teleport": "传送",
+  "gui.tempad.delete": "删除坐标",
+  "gui.tempad.options_header": "选择颜色：",
+  "gui.tempad.timedoor": "任意门",
+  "tooltip.tempad.timeleft": "充至满电所需时间：",
+  "tooltip.tempad.fullycharged": "充电完毕。再次充满电所需时间为%s。",
+  "tooltip.tempad.energy_info": "能量：%s/%s RF",
+  "tooltip.tempad.energy_cost": "每次使用消耗%sRF",
+  "tooltip.tempad.item_option_info": "需要使用可燃物品",
+  "tooltip.tempad.experience_level_cost": "需要%s级经验才能使用任意门",
+  "tooltip.tempad.experience_points_cost": "需要%s点经验才能使用任意门",
+  "item.tempad.tempad": "§6任意门",
+  "item.tempad.he_who_remains_tempad": "§6任意门",
+  "item.tempad.he_who_remains_tempad.tooltip": "“遗留之人”的财产",
+  "advancements.tempad.tempad": "永远\ue908\ue909永远\ue908\ue909",

```

```suggestion
  "advancements.tempad.tempad": "永远……永远……",
```
请直接使用省略号
所有中文文件这一部分均需修改